src/components/UserContext.tsx

```diff
@@ -50,3 +62,53 @@ export const reducer = (state: UserState, action: UserAction): UserState => {
 export const UserStateContext = React.createContext<UserState>(initialState)
 
 export const useUser = () => useContext(UserStateContext)
+
+export default function UserContextComp({ children }) {
```

Comment:
Why don't you use ES6 functions?
it can be replaced by `export default ({ children }) => {`

Reply:
This way in stack trace will show up UserContextComp not default without name. You can see from Dan Abramov's twitter